templates/base/server/.rtag/store.ts.hbs

```diff
@@ -78,3 +78,10 @@ export default class Store {
     }
   }
 }
+
+function ctx() {
+  return {
+    randInt: () => Math.random(),
```

Comment:
is calling `ctx.randInt()` within a state change function supposed to always return the same thing? Currently it looks like it would just call `Math.random()` each time, so you would get a different number. Not sure how this would actually be deterministic.

Reply:
check it out now, im using [seedrandom](https://www.npmjs.com/package/seedrandom).

this will change a bit in the future -- it will assign a random seed for each state during `initialize` and write it down in the replay log. then, if replaying, the seed used will come from the log instead of being random

Reply:
cool, makes a lot more sense now with the new changes. lgtm